src/main/scala/com/monsanto/arch/cloudformation/model/resource/IAM.scala

```diff
@@ -83,9 +83,8 @@ object `AWS::IAM::ManagedPolicy` extends DefaultJsonProtocol {
   implicit val format: JsonFormat[`AWS::IAM::ManagedPolicy`] = jsonFormat8(`AWS::IAM::ManagedPolicy`.apply)
 }
 
-
-case class AWSManagedPolicy(name: String) {
-  def buildARN = s"arn:aws:iam::aws:policy/$name"
+case class AWSManagedPolicy(name: Token[String], accountId : Token[String] = "aws") {
```

Comment:
Yes. This is when there is a policy created, not through cloudformation, but manually. The other approach I could see would be to create another case class and implicit converter and modify ManagedPolicyARN to have a third type (or have the different types extend a sealed trait). thoughts?